.eslintrc.json

```diff
@@ -13,6 +13,12 @@
   },
   "parser": "babel-eslint",
   "rules": {
-    "no-var": ["error"]
+    "no-var": ["error"],
+    "no-empty": [
```

Comment:
Please don't add a global eslint exception. Instead use an `// eslint-disable-next-line no-empty` comment at the place where you want to ignore this rule!